ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -106,9 +106,9 @@ commandName
    ;
 
 searchCommand
-   : (SEARCH)? fromClause                       # searchFrom
-   | (SEARCH)? fromClause logicalExpression     # searchFromFilter
-   | (SEARCH)? logicalExpression fromClause     # searchFilterFrom
+   : (SEARCH)? fromClause                                           # searchFrom
+   | (SEARCH)? fromClause logicalExpression (logicalExpression)*    # searchFromFilter
+   | SEARCH logicalExpression fromClause                            # searchFilterFrom
```

Comment:
The `SEARCH` keyword changed to required since the query `describe source=t` could be ambiguous.
`describe source=t` should throw syntax error "describe source= <==== .."
But it could be matched to
```
(SEARCH)? logicalExpression fromClause
```
logicalExpression -> valueExpr -> fieldExpr -> `describe`
fromClause -> `source=t`